src/jit/gentree.cpp

```diff
@@ -7030,7 +7030,7 @@ GenTreePtr Compiler::gtNewPutArgReg(var_types type, GenTreePtr arg, regNumber ar
     assert(arg != nullptr);
 
     GenTreePtr node = nullptr;
-#if !defined(LEGACY_BACKEND) && defined(_TARGET_ARM_)
+#if !defined(LEGACY_BACKEND) && defined(ARM_SOFTFP)
```

Comment:
If we change this way, I think we can remove the if statement.

Reply:
OK, sounds good.